src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -183,13 +184,14 @@ public void getUsers(String query, String orderBy,
               try {
                 response.write(JSON_MAPPER.writeValueAsString(user), "UTF-8");
               } catch (JsonProcessingException e) {
-                logger.error("This error should never happen: " + e);
+                throw new UncheckedIOException(e);
               }
             }, reply -> {
               if (reply.succeeded()) {
                 response.write(String.format(JSON_USERS_FOOTER, cnt[0], cnt[0]));
               } else {
-                response.setStatusCode(500).setStatusMessage(reply.cause().getLocalizedMessage());
+                logger.error("Get users as a stream failed: " + reply.cause().getMessage(), reply.cause());
```

Comment:
RMB captured and logged it as ERROR.

Reply:
Yes, you are right. I'm removing the duplicate logging.